### PR TITLE
feat: add minutes to meeting model

### PR DESCRIPTION
**Pull request by kobemertens**
https://kanselarij.atlassian.net/jira/software/c/projects/KAS/boards/1?modal=detail&selectedIssue=KAS-3959

### CODE REVIEW

File: app/models/meeting.js [hunk 1]
```diff
@@ -33,7 +33,11 @@ export default class Meeting extends Model {
     async: true,
   })
   internalDocumentPublicationActivity;
-
+  @belongsTo('piece', {
+    inverse: 'meeting',
```
Comment (tdn) [inline app/models/meeting.js]: I see a possible conflict here with https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/e660ce9a47092a46b264d2d824b0b876c51f09c2/app/models/meeting.js#L44

Would rename this (see related app-kaleidos PR). Suggestion: `isMinutesForMeeting`

**Review by tdn — CHANGES_REQUESTED** [review]

File: app/models/meeting.js (hunk 1 above)
Comment (kobemertens) [reply app/models/meeting.js]: Good catch, I will make the necessary changes

**Review by tdn — APPROVED** [review]
Looks like there are some unrelated changes from another branch in here (agenda-detail and agendaitem-controls.js). Be sure to merge those first and maybe check this one again so nothing gets reverted. Maybe rebase this first?

Changes specific to this PR look good

File: app/models/minutes.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import Piece from './piece';
+import { attr, belongsTo } from '@ember-data/model';
+
+export default class Report extends Piece {
```
Comment (ValenberghsSven) [inline app/models/minutes.js]: `class Minutes`

File: app/models/meeting.js [hunk 3]
```diff
@@ -33,6 +33,11 @@ export default class Meeting extends Model {
     async: true,
   })
   internalDocumentPublicationActivity;
+  @belongsTo('minutes', {
+    inverse: 'isMinutesForMeeting',
```
Comment (ValenberghsSven) [inline app/models/meeting.js]: A property/relation starting with 'is', 'has', 'can', etc sounds like a boolean.
perhaps just `minutesForMeeting` ?

**Review by ValenberghsSven — CHANGES_REQUESTED** [review]

File: app/models/meeting.js (hunk 3 above)
Comment (ValenberghsSven) [reply app/models/meeting.js]: @tdn thoughts? Goes against your suggestion

Comment (kobemertens) [reply app/models/meeting.js]: Good point, I will change it

File: app/models/minutes.js (hunk 2 above)
Comment (kobemertens) [reply app/models/minutes.js]: This was fixed in another branch, will fix here as well

Comment (kobemertens) [reply app/models/minutes.js]: fixed in [f8da6c6](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/1737/commits/f8da6c60a6961df6793899f8ab321b7dcc6b6d4b)

File: app/models/meeting.js (hunk 3 above)
Comment (kobemertens) [reply app/models/meeting.js]: fixed in [38c8e0c](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/1737/commits/38c8e0cf6bac0960870e7ea6ff6fe9985bd7efa4)

**Review by ValenberghsSven — APPROVED** [review]

**Pull request merged**